debian/changelog

```diff
@@ -1,3 +1,9 @@
+gnome-software (3.26.0-0) master; urgency=medium
```

Comment:
I see that 3.26.0 is in sid, so we should look and update to that instead of our own packaging.

Reply:
Right. I looked into the 3.25.92 version before the 3.26.0 was out but I found it so different (it now creates a new package for flatpak and some other plugins, as well as the complexity in the rules) that I preferred to just adapt our package (that has been manually updated by us for a long time now).

If there's a problem with the current debian package and/or you prefer nonetheless that I really use the one from upstream, I can do that though it'll take a couple of hours more to have everything tested again.